#issue-203-204 Added test coverage for Class Preference generator and preference di.xml generator

serhiyzhovnir opened the pull request:
**Description** (*)
This PR adds test coverage for `com.magento.idea.magento2plugin.actions.generation.generator.PreferenceDiXmlGenerator` and `com.magento.idea.magento2plugin.actions.generation.generator.PreferenceClassGenerator` generators.

**Fixed Issues (if relevant)**

1. magento/magento2-phpstorm-plugin#203: Integration tests: cover class preference generator
2. magento/magento2-phpstorm-plugin#204: Integration tests: cover preference di.xml generator

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

eduard13 commented on tests/com/magento/idea/magento2plugin/actions/generation/generator/PreferenceClassGeneratorTest.java:
```diff
@@ -0,0 +1,77 @@
+package com.magento.idea.magento2plugin.actions.generation.generator;
```
Could you please add the Copyright annotations here?

eduard13 replied:
```suggestion
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
package com.magento.idea.magento2plugin.actions.generation.generator;
```

eduard13 commented on tests/com/magento/idea/magento2plugin/actions/generation/generator/PreferenceDiXmlGeneratorTest.java:
```diff
@@ -0,0 +1,112 @@
+package com.magento.idea.magento2plugin.actions.generation.generator;
```
Here as well

eduard13 replied:
```suggestion
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
package com.magento.idea.magento2plugin.actions.generation.generator;
```

serhiyzhovnir commented:
Hi @eduard13  @VitaliyBoyko 
I fixed all errors with static tests and also I had added the excluding for the [category/java/documentation.xml/CommentSize](https://pmd.github.io/latest/pmd_rules_java_documentation.html#commentsize) rule. This rule throws the error if a method docs comment has more then 6 lines, it's a problem for methods with many arguments.
For this PR it throws the error on this method comment: https://github.com/magento/magento2-phpstorm-plugin/pull/238/files#diff-dc18067fb134408559fa7f2888c01ad8R63-R73.
Could you review this one, please?
Thank you!

VitaliyBoyko commented:
Seems I can't merge it without @eduard13 
Let's wait for him back from the vacation 😄
